tests/TestCase.php

```diff
@@ -22,7 +23,13 @@ public function setUp(): void
     {
         parent::setUp();
 
-        $this->withFactories(__DIR__.'/../database/factories');
+        FactoriesFactory::guessFactoryNamesUsing(function ($name) {
+            $modelName = (string) '\\IndieHD\\Velkart\\Database\\Factories\\'.
+                (class_basename($name)).
+                'Factory';
+
+            return $modelName::new();
+        });
```

Comment:
This is clever! How did you even discover that this method exists in the `Illuminate\Database\Eloquent\Factories\Factory` class?

Reply:
I clicked go to source in my editor and checked the class :)

I'm wondering if it should just return the class name and not the `::new()` thing. 

Will have to check.

Reply:
Not sure why it worked, but it should really be with just the class name returned. Will create a bug PR

Reply:
Indeed. I wonder how this worked before. I just want to ensure that it's "doing what we think", but we can take that conversation over to the new PR.